src/Integration.Vsix/CFamily/PortedFromJava/MsvcDriver.cs

```diff
@@ -260,7 +273,8 @@ public static Request ToRequest(CFamilyHelper.Capture[] captures)
                     }
                     else
                     {
-                        string file = Absolute(capture.Cwd, arg);
+                        var relativeOrAbsolutePath = arg.Replace("\"", "");
+                        string file = Absolute(capture.Cwd, relativeOrAbsolutePath);
```

Comment:
If you look at the definition of absolute, there are asserts and sometimes it throws. I think it would be better if we catch the exception and log and the exact user option that lead to this exception( it will help us in debugging) and then we simply skip the option instead of letting the exception propagate(this what MSVC does when it faces invalid additional options it just ignores it)

Reply:
@abbas-sabra-sonarsource Absolute is called in other places as well, did you mean to add the try-catch only in case of this "else" block, or in all the cases?

Reply:
It's better to do it for all 3 calls since all of them can suffer from the same problem. 
It might be a better idea to do it inside the function `Absolute` and then print the "file path" that leads to this exception and return null(that is ignored in the three cases)

Reply:
Great, thanks 👍

Reply:
Just double-checked the 3rd call should be safer since it cannot be added from additional options(it is coming from env).